recipes/cpputest/all/conanfile.py

```diff
@@ -0,0 +1,100 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+class CppUTestConan(ConanFile):
+    name = "cpputest"
+    description = \
+"CppUTest is a C /C++ based unit xUnit test framework for unit testing and for test-driving your code." \
+"It is written in C++ but is used in C and C++ projects and frequently" \
+"used in embedded systems but it works for any C/C++ project."
+    license = "BSD-3-Clause License"
```

Comment:
make sure to follow the SDPX identifiers

```suggestion
    license = "BSD-3-Clause"
```